src/main/java/org/gridsuite/study/server/elasticsearch/EquipmentInfosServiceMock.java

```diff
@@ -32,21 +32,41 @@ public TombstonedEquipmentInfos addTombstonedEquipmentInfos(TombstonedEquipmentI
         return tombstonedEquipmentInfos;
     }
 
+    @Override
+    public void addAllEquipmentInfos(List<EquipmentInfos> equipmentsInfos) {
```

Comment:
default implementation on interface EquipmentInfosService ?

Reply:
This is not really a default implementation, it is a specific implementation for EquipmentInfosServiceMock (which in fact does nothing).